Sources/Honeycomb/UIKit/NavigationInstrumentation.swift

```diff
@@ -0,0 +1,18 @@
+#if canImport(UIKit)
+    import Foundation
+    import OpenTelemetryApi
+    import UIKit
+
+    private let honeycombInstrumentationView = "@honeycombio/instrumented-view"
```

Comment:
PR #20 is probably gonna go in before this one, and has `"@honeycombio/instrumentation-view"`. Let's find a place where we can put it as a constant and reuse it. Maybe all of our tracer names should be in one file? 🤔